storefront/styles/main.css

```diff
@@ -3491,6 +3491,14 @@ input.num-product::-webkit-inner-spin-button {
   min-height: 100%;
 }
 
+.product-link {
+  color: #66a8a6 !important;
+}
+
+.product-link:hover {
+  color: #273ac8 !important;
+}
```

Comment:
We should move these CSS to another file so we can easily maintain them in the future.

Reply:
I fixed it. Please review again, thanks.